### DISCUSSION TITLE
Web Scrape Using Observable

### DISCUSSION THREAD

**Discussion by thegilpatrick** [quarto-dev/quarto-cli#5974] [Show and Tell]
### Description

Wrote a blog post about web scraping recipes from cooking websites try it out.

https://thegilpatrick.com/posts/recipe-web-scrape/

**Comment by Charlotte-br560** [comment]
Thanks for sharing your blog post! It's always great to see interesting projects like this. I'll definitely check it out. If you're interested in further optimizing your web scraping workflow, you might find [Crawlbase](https://crawlbase.com/) helpful. Keep up the good work!
